### ISSUE TITLE
Local storage contains lots of files with no bookmarks

### ISSUE THREAD

**Issue by dakaraphi** [alefragnani/vscode-bookmarks#51]
Local storage of bookmarks contains many files with no bookmarks.  For large projects, this list continue to grow large.

**Comment by alefragnani** [comment]
Hi @dakaraphi ,

Do you mean the size of `file__0.localstorage` file? How could I see the _size_ it occupies?

But yes, each opened file is added to the _List of Bookmarks_, and while saving the session, all files will be there, even if has no bookmarks. I didn't note any performance issues until now, even for large projects, but if you have any report, I could take a look to optimize that.

Hope this helps

**Comment by dakaraphi** [comment]
@alefragnani It was really just an observation.  It wasn't causing me any real problems, other than it made it more difficult to find actual files with bookmarks.  I needed to manually remove an entry to a file I deleted that was causing problems for which I opened the other issue.

I didn't see a purpose for having lots of empty entries, so thought it might have been an oversight and over a long period of time I don't know if it could cause issues.  Just mentioned it for awareness.
